src/index.html

```diff
@@ -0,0 +1,127 @@
+</head>
+<body>
+
+<nav class="navbar">
+  <div class="navbar-brand">
+    <div class="navbar-item">
+      <div class="field has-addons">
+        <div class="control">
+          <input class="input username" type="text" placeholder="enter github username">
+        </div>
+        <div class="control">
+          <button class="button is-info load-username">
+            Load
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <div class="navbar-item">
+      <div class="loader is-hidden" id="spinner"></div>
+    </div>
+
+  </div>
+</nav>
+
+
+<div class="container">
+  <div class="columns is-1-tablet">
+    <div class="profile-container column is-one-third">
+
+      <h2 class="subtitle is-4">Profile</h2>
+
+      <div class="profile">
+        <div class="media">
+          <div class="media-left">
+            <figure class="media-left image is-64x64">
+              <img src="http://placekitten.com/200/200" id="profile-image">
+            </figure>
+          </div>
+          <div class="media-content">
+            <p class="title is-5" id="profile-name">John Smith</p>
+            <p class="subtitle is-6"><a href="#" id="profile-url">@johnsmith</a></p>
+          </div>
+        </div>
+
+
+        <div class="content" id="profile-bio">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, eu pellentesque tortor
+            vestibulum ut.</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="events-container column">
+      <h2 class="subtitle is-4">History</h2>
+
+      <div class="timeline" id="user-timeline">
+
+        <div class="timeline-item">
+          <div class="timeline-marker"></div>
+          <div class="timeline-content">
+            <p class="heading">Jan 21, 2016</p>
+            <div class="content">
+                  <span class="gh-username">
+                    <img src="https://avatars.githubusercontent.com/u/2310778?"/>
+                    <a href="https://github.com/sethii">sethii</a>
+                  </span>
+              closed
+              <a href="https://github.com/TheSoftwareHouse/Kakunin/pull/41">pull request</a>
+              <p class="repo-name">
+                <a href="https://github.com/TheSoftwareHouse/Kakunin">TheSoftwareHouse/Kakunin</a>
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="timeline-item is-primary">
+          <div class="timeline-marker is-primary"></div>
+          <div class="timeline-content">
+            <p class="heading">Jan 19, 2016</p>
+            <div class="content">
+                <span class="gh-username">
+                  <img src="https://avatars.githubusercontent.com/u/1208786?"/>
+                  <a href="https://github.com/simonsipl">gajdus</a>
+                </span>
+              created
+              <a href="https://github.com/TheSoftwareHouse/Kakunin/pull/41#discussion_r176273897">comment</a>
+              to
```

Comment:
it is wise to have all html as a child node. In this case would do no harm if you could tag ```created``` by <span> or any other elem. This create a mess when is as plaintext, when you try to moderate it via JS.
https://developer.mozilla.org/pl/docs/Web/API/Element/innerHTML
vs
https://developer.mozilla.org/pl/docs/Web/API/Element/childNodes